copter/source/docs/safety-multicopter.rst

```diff
@@ -196,27 +196,27 @@ MultiCopter Safety
    #. Gusting winds around objects can also worsen the problem.
    #. Radio signal loss can also occur.
    #. If your copter is approaching a potentially interfering object,
-      immediately switch to stabilize mode and land or retrieve the
+      immediately switch to Stabilize mode and land or retrieve the

```

Comment:
For modes I actually like to capitalise fully if possible (ie mode is AUTO not Auto or auto - ditto for other modes like STABILIZE ... ).